client/src/components/PlantButton/stateUtils.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+    controls: {
+        water: "water",
+    },
+    plantImageUrls: [
+        "https://i.ibb.co/DVP9kkq/Tree1.png",
+        "https://i.ibb.co/CnQgYZp/Tree2.png",
+        "https://i.ibb.co/2s4TDYc/Tree3.png",
+        "https://i.ibb.co/5r5tqWY/Tree4.png",
```

Comment:
Is there are reason why we don't include the images in the repository directly? Perhaps there should be an `images` folder inside the `Plant` folder that contains the images. I am worried that these i.ibb links might expire and then we will not be able to see the images.